Lib/fontTools/ttLib/tables/T_S_I__0.py

```diff
@@ -3,7 +3,7 @@
 from . import DefaultTable
 import struct
 
-tsi0Format = '>HHl'
+tsi0Format = '>HHL'
```

Comment:
Is there a somewhere a spec for the table format? If so, it might be good to add its URL here (couldn’t find anything on the web, but maybe I’m using the wrong search engine). Otherwise, mind adding a comment that explains the purpose of the table?